database/sqlite/sqlite_aclk_alert.c

```diff
@@ -45,24 +16,31 @@ void update_filtered(ALARM_ENTRY *ae, uint32_t unique_id, char *uuid_str) {
     ae->flags |= HEALTH_ENTRY_FLAG_ACLK_QUEUED;
 }
 
-#define SQL_SELECT_ALERT_BY_UNIQUE_ID "SELECT hl.unique_id FROM health_log_%s hl, alert_hash ah WHERE hl.unique_id = %u " \
-                            "AND hl.config_hash_id = ah.hash_id " \
+#define SQL_SELECT_ALERT_BY_UNIQUE_ID "SELECT hl.unique_id FROM health_log hl, alert_hash ah WHERE hl.unique_id = %u " \
+                            "AND hl.config_hash_id = ah.hash_id AND host_id = @host_id " \
```

Comment:
Why are we not using an `INNER JOIN` ? I will write without to test the new query:

```sh
SELECT hl.unique_id FROM health_log hl INNER JOIN alert_hash ah ON(hl.config_hash_id = ah.hash_id) WHERE hl.unique_id = %u  AND host_id = @host_id;
```

Reply:
It is ok to use the older syntax : from the sqlite documentation https://www.sqlite.org/optoverview.html#joins
```
The ON and USING clauses of an inner join are converted 
into additional terms of the WHERE clause prior to WHERE 
clause analysis described [above in paragraph 2.0]
Thus with SQLite, there is no computational advantage to 
use the newer SQL92 join syntax over the older SQL89 comma-join syntax. 
They both end up accomplishing exactly the same thing on inner joins
```

Reply:
Thank you to bring this additional info.